### DISCUSSION TITLE
how to reset account password?

### DISCUSSION THREAD

**Discussion by plimeor** [planetscale/discussion#338] [FAQs] [answered]
https://auth.planetscale.com/password/new will redirect to home page, so there is another way to reset account password?

**Comment by fooforge** [comment, marked as answer]
Hey @plimeor,

That URL will only work for logged-out users and redirect you to home if logged in. Try https://app.planetscale.com/settings/account instead.

If you don't have your password anymore, open https://auth.planetscale.com/password/new in a private browser session to verify your email and reset your password.
